core/src/main/java/org/sonarsource/sonarlint/core/container/model/DefaultRuleDetails.java

```diff
@@ -30,13 +33,16 @@
   private final String name;
   private final String htmlDescription;
   private final String severity;
+  private final String type;
   private final String extendedDescription;
 
-  public DefaultRuleDetails(String key, String name, String htmlDescription, String severity, String language, Set<String> tags, String extendedDescription) {
+  public DefaultRuleDetails(String key, String name, String htmlDescription, String severity, @Nullable String type, String language, Set<String> tags,
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Constructor has 8 parameters, which is greater than 7 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS00107)